.gitignore

```diff
@@ -39,3 +39,9 @@ mprofile_*.dat
 
 # Ignore Windows artefacts
 *.dll
+
+# Ignore virtualenv
+.venv
```

Comment:
Actually, I create my venv under `dist` to avoid having to exclude one more file. Would you mind?